test_Lilit_Sargsyan.py

```diff
@@ -0,0 +1,2 @@
+def test_print_name(name="Lilit",surname="Sargsyan",nickname="sarly"):
+    print(f"My name is {name} {surname}, and my nickname is {nickname}")
```

Comment:
Add description to your nickname, so describe meaning of the name and print it.

Reply:
Added description of my nickname and the meaning of my name